### ISSUE TITLE
Accessibility - Country code selector doesn't work with screen readers

### ISSUE THREAD

**Issue by AlanFors** [jackocnr/intl-tel-input#777]
Using a screen reader such as Narrator, NVDA, or JAWS as a low-vision or blind user would, the country code selector does not expand, and choices are not navigable using keystrokes.
Blind user will tab through the page, and land on the country selector. It does not announce that there is a list to choose from, so user does not know what action to perform. If user was aware this is a dropdown list, the control does not behave properly - Enter, or Space should open the drop-down, and down-arrow should navigate the list of items and allow selection.

The control, as is, operates properly when not using a screen reader, but when using one there is no way to select from the list via keyboard.

I've worked around this by applying aria attributes via javascript:

`    <script type="text/javascript">

        $(document).ready(function () {
            $('.flag-container').attr('aria-label', 'Country code');
            $('.selected-flag').attr('role', 'button');
            $('.selected-flag').attr('aria-haspopup', 'true');
            $('.country').attr('tabindex', '-1');
        });
    </script>
`

Also, note the aria-label to announce what this control is providing. This way will be problematic for internationalization, but it's better than nothing.

**Comment by jackocnr** [comment]
Hi there, thanks for reporting this. Can I ask: are you using the latest version? v13.0.2 at the time of writing. We just recently added a bunch of aria attributes - if you search the [source code](https://github.com/jackocnr/intl-tel-input/blob/master/src/js/intlTelInput.js) for `aria-` you'll see lots of results.

We're currently setting `role="combobox"` on the selected flag element, and instead of using `aria-haspopup` attribute, we're using `aria-owns` to tie it to the country list. Is that not right?

If you're using the latest version and you still think we need to make the changes you listed, please can you explain why we need each attribute (in addition to what we currently have)? Thank you.

**Comment by AlanFors** [comment]
No, I'm using v12.4.0. I'll try the latest version at some point, but this will have to wait while I drive through a release of my product.

**Comment by jackocnr** [comment]
Closing due to lack of activity.
